src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -821,11 +826,16 @@
   - name: --enable-encryption-at-host
     type: bool
     short-summary: Enable EncryptionAtHost, default value is false.
+  - name: --enable-ultra-ssd
+    type: bool
```

Comment:
recommend to use get_three_state_flag() for boolean https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags